elements/lisk-p2p/test/utils/network_setup.ts

```diff
@@ -15,11 +15,13 @@
 import { P2P } from '../../src/index';
 import { wait } from './helpers';
 import { platform } from 'os';
+import {
+	DEFAULT_MAX_OUTBOUND_CONNECTIONS,
+	DEFAULT_MAX_INBOUND_CONNECTIONS,
+} from '../../src';
 export const NETWORK_START_PORT = 5000;
 export const NETWORK_PEER_COUNT = 10;
 export const POPULATOR_INTERVAL = 50;
-export const DEFAULT_MAX_OUTBOUND_CONNECTIONS = 20;
-export const DEFAULT_MAX_INBOUND_CONNECTIONS = 100;
 export const DEFAULT_CONNECTION_TIMEOUT = 500;
```

Comment:
What about this `DEFAULTS`? Shall we maybe rename or add them to `src/constants.ts`?

Reply:
All the other values are actually not defaulted constant values but have been changed to run tests faster. I can rename it and remove `DEFAULT` for the ones that are not coming from constants